src/test/kotlin/com/hibob/academy/unitests/keren/UserServiceTest.kt

```diff
@@ -0,0 +1,110 @@
+package com.hibob.academy.unitests.keren
+
+import com.hibob.bootcamp.unittests.*
+import org.hibernate.validator.internal.util.Contracts.assertTrue
+import org.junit.jupiter.api.Assertions.assertEquals
+import org.junit.jupiter.api.Assertions.assertFalse
+import org.junit.jupiter.api.Test
+import org.junit.jupiter.api.assertThrows
+import org.mockito.kotlin.*
+
+class UserServiceTest{
+    val token = "Waffle"
+    val id: Long= 123
+    val name: String = "Ron Azar"
+    val email: String = "ron.azar@hibob.io"
+    val password: String ="12345"
+    val isEmailValid = true
+
+    private val user = User(id,name, email ,password, isEmailValid)
```

Comment:
Instead of defining all parameters you can define them directly here
    private val user = User(123,"Ron Azar", "ron.azar@hibob.io" ,"12345", true)

Reply:
Your'e absolutely right